config/gregtech/worldgen/nether/topaz_vein.json

```diff
@@ -1,32 +1,32 @@
-{
-  "weight": 40,
-  "density": 0.3,
-  "max_height": 90,
-  "min_height": 40,
-  "dimension_filter": ["is_nether"],
-  "generator": {
-    "type": "ellipsoid",
-    "radius": [5, 5]
-  },
-  "filler": {
-    "type": "simple",
-    "value": {
-      "type": "weight_random",
-      "values": [
-        {
-          "weight": 55,
-          "value": "ore:topaz"
-        },
-        {
-          "weight": 40,
-          "value": "ore:blue_topaz"
-        }
-      ,
-        {
-          "weight": 5,
-          "value": "ore:zirkelite"
-        }
-      ]
-    }
-  }
+{
+  "weight": 40,
+  "density": 0.3,
+  "max_height": 90,
+  "min_height": 40,
+  "dimension_filter": ["is_nether"],
+  "generator": {
+    "type": "ellipsoid",
+    "radius": [5, 5]
+  },
+  "filler": {
+    "type": "simple",
+    "value": {
+      "type": "weight_random",
+      "values": [
+        {
+          "weight": 55,
+          "value": "ore:topaz"
+        },
+        {
+          "weight": 40,
+          "value": "ore:blue_topaz"
+        }
+      ,
+        {
+          "weight": 5,
+          "value": "ore:zirkelite"

```

Comment:
Zirkelite is actually a really powerful ore. It contains Calcium, Thorium, Cerium (1 source of carbon fibers), zirconium, rutile, and niobium. While you do need 56 dusts, this might be slightly too good? I'm conflicted. The very low weight is probably enough to offset that. I'll leave it to your judgement.

Reply:
I think with a weight this low it's fine